Formula/mingw-w64.rb

```diff
@@ -0,0 +1,177 @@
+class MingwW64 < Formula
+  desc "Minimalist GNU for Windows and GCC cross-compilers"
+  homepage "https://mingw-w64.org/"
+  url "https://downloads.sourceforge.net/project/mingw-w64/mingw-w64/mingw-w64-release/mingw-w64-v5.0.1.tar.bz2"
+  sha256 "9bb5cd7df78817377841a63555e73596dc0af4acbb71b09bd48de7cf24aeadd2"
+
+  # Multilib does not work currently
+  # option "without-multilib", "Build 64-bit compiler with multilib support"
```

Comment:
Probably worth removing this comment/option and all references to it until it's working.